views/MainView.tsx

```diff
@@ -206,6 +206,12 @@ const MainView = () => {
     t("lang")
   );
   const [region, setRegion, clearRegion, regionReady] = useStringMmkv(Key.Region, t("region"));
+  useEffect(() => {
```

Comment:
Actually, you have to check `regionReady` before you can update the current region value. I think you can put it before L325 after the migration.

https://github.com/zhxie/conch-bay/blob/1a129927496968361ba7a4a589e7c8d59603021a/views/MainView.tsx#L324-L326

Reply:
Updated [0c53747](https://github.com/zhxie/conch-bay/pull/40/commits/0c5374756287b0839add17051ad9dd8aad2e7fd0)